src/slider/slider.js

```diff
@@ -5,200 +5,141 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
+
 import * as React from 'react';
-import {findDOMNode} from 'react-dom';
-import {thumbWidth} from './constants.js';
-import type {StatelessStateT, PropsT} from './types.js';
+import {Range} from 'react-range';
+import type {PropsT} from './types.js';
 import {
   Root as StyledRoot,
-  Axis as StyledAxis,
-  AxisRange as StyledAxisRange,
+  Track as StyledTrack,
+  InnerTrack as StyledInnerTrack,
   Tick as StyledTick,
   TickBar as StyledTickBar,
   Thumb as StyledThumb,
+  InnerThumb as StyledInnerThumb,
+  ThumbValue as StyledThumbValue,
 } from './styled-components.js';
 import {getOverrides} from '../helpers/overrides.js';
-/* global document */
-/* global window */
-
-function getTickValue(tick: number | {value: number}): number {
-  if (typeof tick === 'object') {
-    return tick.value;
-  }
-
-  return tick;
-}
 
-function getTickLabel(tick: number | {label: React.Node}): React.Node {
-  if (typeof tick === 'object') {
-    return tick.label;
+// values.length should not be bigger than two
+// because our design doesn't support more than
+// two thumbs
+const limitValues = (values: number[]) => {
+  if (values.length > 2 || values.length === 0) {
```

Comment:
Possibly could do `values` props not required and set the default to `[min, min]` or `[min, max]`

Reply:
It would not be much helpful here since the component is stateless, so if you omitted the `values` prop you couldn't move the slider. What's the use-case for that? Maybe the default value would make sense in the `StatefulSlider` component?

Reply:
Oh, you're totally right. It can only be applicable to the stateful slider.

Reply:
default (single mid-point) added to StatefulSlider